native/acter/api.rsh

```diff
@@ -1117,6 +1123,37 @@ object MsgContent {
     fn url_previews() -> Vec<UrlPreview>;
 }
 
+object MembershipContent {
+    /// The ID of the user whose profile changed.
+    fn user_id() -> UserId;
+
+    /// The membership change induced by this event.
+    fn change() -> string;
+}
```

Comment:
I am confused. Why not use the existing API and `MembershipChange`-Object, extend the change as it is here and expose the `ProfileChange` on that? I find the previous API easier to understand and follow.

Reply:
In future PR, I am going to implement room state as following.
```rs
pub enum TimelineEventContent {
    Message(MsgContent),
    MembershipChange(MembershipContent),
    ProfileChange(ProfileContent),
    RoomCreate(RoomCreateContent),
    RoomName(RoomNameContent),
    ...
}
```
And will append the following apis.
```
    /// get the content of room create event
    fn room_create() -> Option<RoomCreateContent>;

    /// get the content of room name event
    fn room_name() -> Option<RoomNameContent>;
```